kolibri/core/templates/kolibri/base.html

```diff
@@ -18,10 +15,11 @@
 </ul>
 {% endblock %}
 
-<hr>
-
-{% block content %}
-
+<main></main>
+{% block frontend_assets %}
```

Comment:
We should hold off any rendering through Vue if the document is not ready yet, not put our script tags in the middle of our HTML.

http://stackoverflow.com/questions/436411/where-is-the-best-place-to-put-script-tags-in-html-markup

We can add the async directives later, if we want to optimize page load, but this functionality should be implemented by listening for a documentContentLoaded event.